data/data_generator.py

```diff
@@ -211,33 +211,47 @@ def generate_weather(self,
         return df
             
     def generate_prices(self,
+                      start_day_time = '08:00:00',
+                      end_day_time = '17:00:00',
                       price_constant = 0.2,
                       price_day = 0.3,
-                      price_night = 0.1,
-                      start_day_time = '08:00:00',
-                      end_day_time = '17:00:00'):
+                      price_night = 0.1, 
+                      price_district_heating_power = 0.1,
+                      price_gas_power = 0.07,
+                      price_biomass_power = 0.2,
+                      price_solar_thermal_power = 0.0):
+                      
         '''Append the prices for different energy sources.
         There are three different scenarios considered for electricity:
             1. PriceElectricPowerConstant: completely constant price
             2. PriceElectricPowerDynamic: day/night tariff
             3. PriceElectricPowerHighlyDynamic: spot price 
             
-        All prices are expressed in ($/euros)/Kw*h.
+        All prices are expressed in ($/euros)/KW*h.
```

Comment:
Tiny detail: maybe kW*h more correct? sometimes I mistake a capital K for the units.

Reply:
I edited so that everything reads kWh.